webapp/channels/src/components/linking_landing_page/linking_landing_page.tsx

```diff
@@ -241,11 +241,18 @@ export default class LinkingLandingPage extends PureComponent<Props, State> {
         if (this.state.redirectPage) {
             return (
                 <div className='get-app__download-link'>
-                    <FormattedMarkdownMessage
-                        id='get_app.openLinkInBrowser'
-                        defaultMessage='Or, [open this link in your browser.](!{link})'
+                    <FormattedMessage
+                        id='getApp.downloadLinkInBrowser'
+                        defaultMessage='Or, <a>open this link in your browser</a>.'
                         values={{
-                            link: this.state.location,
+                            a: (chunks: string) => (
+                                <ExternalLink
+                                    href={this.state.location}
+                                    location='get_app'
```

Comment:
Lets change this location to `landingPage`

Reply:
Done!